build.yaml

```diff
@@ -823,7 +823,7 @@ steps:
       cd /io/repo/hail
       chmod 755 ./gradlew
       time retry ./gradlew --version
-      export SPARK_VERSION="3.0.2" SCALA_VERSION="2.12.10"
+      export SPARK_VERSION="3.3.0" SCALA_VERSION="2.12.15"
```

Comment:
Let's not change this. We have to go through and carefully check the HDInsight settings to make sure that we're starting clusters compatible with Spark 3.3.0. When I last went through the pain of getting HDInsight configured correctly, this was the only compatible version of HDInsight.

Reply:
rolled back changes to this file 👍